### ISSUE TITLE
strictQueryParams:false with trailingSlash:true doesn't seem to work

### ISSUE THREAD

**Issue by acolby** [router5/router5#124]
Love the project!

It seems that there is a path matching issue when strictQueryParams:false and trailingSlash:true. I've tried with v4.5.1 and v4.4.1.

Please let me know if I'm missing something. If not, I'd be happy to dig in and toss up a PR.

To Re-Pro

```
var router5 = require('router5');

var routes = [
  { name: 'root', path: '/', title: 'Home Page' },
  { name: 'home', path: '/home', title: 'Home Page' }
]

var router = router5.createRouter(routes, { defaultRoute: 'page404', trailingSlash: true, strictQueryParams: false });

console.log(router.matchPath('/'));
console.log(router.matchPath('/?s=3'));

console1 =
{ name: 'root',
params: {},
path: '/',
meta: { params: { root: {} }, id: 1 } }

console2 =
null
```

Thanks in advance for any help!

**Comment by troch** [comment]
Hey, I will try to reproduce and come back you.

**Comment by troch** [comment]
Fixed with v4.5.2. Sorry for the time it took, and thanks for your support!

**Comment by acolby** [comment]
Are you kidding, that was fast. Thank you for all the effort you've put into this project!!
